test/mod/test_util_experimentalflags.py

```diff
@@ -0,0 +1,39 @@
+#
+# Test for the util.experimentalflags
+#
+import pytest
+
+from osbuild.util import experimentalflags
+
+
```

Comment:
Not a big deal, but I notice there aren't any mixed type test strings. eg. `foo=true,bar=baz` and `foo,key=val,bar=false` for example.

Reply:
Excellent point! I will do a small followup for this!